src/pages/people/tabs/__tests__/Wanted.spec.tsx

```diff
@@ -73,6 +77,49 @@ describe('Wanted Component', () => {
     });
   });
 
+  test('Correct calls are made when boxes are clicked', async () => {
+    const userBounty = { ...mockBounties[0], body: {} } as any;
+    userBounty.body = {
+      ...userBounty.bounty,
+      owner_id: person.owner_pubkey,
+      title: 'test bounty here',
+      description: 'custom ticket for testing'
+    };
+
+    (usePerson as jest.Mock).mockImplementation(() => ({
+      person: {},
+      canEdit: false
+    }));
+
+    (useStores as jest.Mock).mockReturnValue({
+      main: {
+        getPersonCreatedBounties: jest.fn(() => [userBounty])
+      },
+      ui: {
+        selectedPerson: '123',
+        meInfo: {
+          owner_alias: 'test'
+        }
+      }
+    });
+
```

Comment:
@okhot please assert that clicking on the checkboxes triggers a getCreatedBounties call